### PR TITLE
Fixes for review 

### PR DESCRIPTION
**Pull request by daron666**
Fixes for a review from PR #174

### CODE REVIEW

File: stracer/src/main/scala/com/evolutiongaming/random/ThreadLocalRandom.scala [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
+    val random = Random.State.fromClock[F]()
+    for {
+      random <- ThreadLocalOf[F].apply(random)
+    } yield {
+      apply(random)
+    }
+
+  }
+
+  def sync[F[_]: Sync: Clock: ThreadLocalOf]: F[Random[F]] =
```
Comment (t3hnar) [inline stracer/src/main/scala/com/evolutiongaming/random/ThreadLocalRandom.scala:20]: please name it `of + 1` = `of1` - basically incrementing version of API, this is better than making up random names :)

File: stracer/src/main/scala/com/evolutiongaming/stracer/util/Probability.scala [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  val default = new Probability {
+    override def apply(sampling: Option[Sampling]): Double = sampling match {
+      case Some(Accept) => alwaysProbabilityValue
+      case Some(Debug) => alwaysProbabilityValue
+      case Some(Deny) => defaultProbabilityValue
+      case None => defaultProbabilityValue
+    }
+  }
+
+  val always = new ConstProbability(alwaysProbabilityValue)
```
Comment (t3hnar) [inline stracer/src/main/scala/com/evolutiongaming/stracer/util/Probability.scala:26]: always define return type for public members

File: stracer/src/main/scala/com/evolutiongaming/stracer/util/Probability.scala [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+  val never = new ConstProbability(neverProbabilityValue)
+
+  class ConstProbability(value: Double) extends Probability {
```
Comment (t3hnar) [inline stracer/src/main/scala/com/evolutiongaming/stracer/util/Probability.scala:30]: private please

Comment (t3hnar) [reply stracer/src/main/scala/com/evolutiongaming/stracer/util/Probability.scala:30]: and maybe just simply name `Const`

File: stracer/src/main/scala/com/evolutiongaming/stracer/util/Probability.scala [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+import com.evolutiongaming.stracer.Sampling
+import com.evolutiongaming.stracer.Sampling._
+
+trait Probability {
+  def apply(sampling: Option[Sampling]): Double
+}
+
+object Probability {
+
+  val defaultProbabilityValue = 0.01
```
Comment (t3hnar) [inline stracer/src/main/scala/com/evolutiongaming/stracer/util/Probability.scala:12]: all private please

File: stracer/src/main/scala/com/evolutiongaming/stracer/util/Probability.scala [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  val alwaysProbabilityValue = 1.0
+  val neverProbabilityValue = -1.0
```
Comment (t3hnar) [inline stracer/src/main/scala/com/evolutiongaming/stracer/util/Probability.scala:14]: how this is supposed to work?

Comment (daron666) [reply stracer/src/main/scala/com/evolutiongaming/stracer/util/Probability.scala:14]: we are checking that current probability is higher than generated double value. double value always generated in range [0.0, 1.0), so -1.0 will always be considered as 'not pass' for traces

File: stracer/src/main/scala/com/evolutiongaming/stracer/util/Probability.scala (hunk 3 above)
Comment (daron666) [reply stracer/src/main/scala/com/evolutiongaming/stracer/util/Probability.scala]: added method `def const(value: Double): Probability`

**Pull request merged**